avahi-common/timeval.c

```diff
@@ -73,20 +85,44 @@ struct timeval* avahi_timeval_add(struct timeval *a, AvahiUsec usec) {
     return a;
 }
 
-AvahiUsec avahi_age(const struct timeval *a) {
-    struct timeval now;
+struct AvahiTimeVal* avahi_now(struct AvahiTimeVal *now) {
```

Comment:
Yes. It would be really confusing if AvahiTimeVal could container either Montonic or some other clock.... which is why there is just the one API and it returns the "Avahi clock". It's always consistent  as long as you are using the AvahiTimeVal